packages/ui/client/pages/index.vue

```diff
@@ -42,12 +42,14 @@ const resizingMain = useDebounceFn((event: { size: number }[]) => {
 function resizeMain() {
   const width = window.innerWidth
   const panelWidth = Math.min(width / 3, 300)
-  mainSizes.value[0] = (100 * panelWidth) / width
-  mainSizes.value[1] = 100 - mainSizes.value[0]
-  recordMainResize([
-    { size: mainSizes.value[0] },
-    { size: mainSizes.value[1] },
-  ])
+  if (browserState) {
```

Comment:
Why did it suddenly stop recording the main resize? `recordMainResize`

Reply:
We drop the middle panel in the default ui

Reply:
But we don't drop it in the browser

Reply:
Are you saying it is ok to break the browser UI?

Reply:
No, I need to test the changes in the browser ui, this change just fix the ui.

This shouldn't break the browser ui, the logic is the same, we only initilize the ui panels properly.

The problem is about using the same storage key for both, the ui and the browser ui.